build/ghost-buster/start.bash

```diff
@@ -35,7 +41,7 @@ if [[ -d "$OVERRIDE/$THEMES" ]]; then
   done
 fi
 
-sed -i -e "s|http://my-ghost-blog.com|${BLOG_DOMAIN}|g" /ghost/config.js
+sed -i -e "s|${DEFAULT_BLOG_DOMAIN}|${BLOG_DOMAIN}|g" /ghost/config.production.json
```

Comment:
When you build container personally for yourself, you may pass your blog url as `--url` option of `ghost install` (see Dockerfile). This option is required. I left the default blog domain for using it in public image. You may set your real domain on start container by passing `BLOG_DOMAIN` env variable, like before.